cmd/kyma/completion/cmd.go

```diff
@@ -15,7 +15,46 @@ func NewCmd() *cobra.Command {
 		Use:   "completion bash|zsh",
 		Short: "Generates bash or zsh completion scripts.",
 		Long: `Use this command to display the shell completion code used for interactive command completion. 
-To configure your shell to load completions, add ` + "`. <(kyma completion bash)`" + ` to your bash profile or ` + "`. <(kyma completion zsh)`" + ` to your zsh profile.
+To configure your shell to load completions, use:
+
+Bash:
+
+  $ source <(kyma completion bash)
+
+  # To load completions for each session, execute once:
+  # Linux:
+  $ kyma completion bash > /etc/bash_completion.d/kyma
+  # macOS:
+  $ kyma completion bash > /usr/local/etc/bash_completion.d/kyma
+
+Zsh:
+
+  $ source <(kyma completion zsh)
+
+  # To load completions for each session, execute once:
+  $ kyma completion zsh > "${fpath[1]}/_kyma"
+
+  # You will need to start a new shell for this setup to take effect.
+
+  # If shell completion is not already enabled in your environment,
+  # you will need to enable it.  You can execute the following once:
```

Comment:
```suggestion
  # you must enable it. Execute the following once:
```

"can" (it's not optional right?) and "will" are not needed. "must" is simpler than "need to" :D